torch/csrc/jit/passes/utils/subgraph_utils.h

```diff
@@ -23,11 +23,20 @@ TORCH_API Node* createSingletonSubgraph(Node* n, Symbol subgraphKind);
 // Merge a node into a subgraph node. If `toMerge` is also a subgraph, the
 // subgraphs are merged.
 // `toMerge` is destroyed.
+// An optional argument 'vmap' could be used to retrieve value mappings.
```

Comment:
Could you specify which direction does the map go in ?

Reply:
The map is old_values -> new_values. The old values are destroyed as we're merging a node and what this map allows us to do is to locate their existing counterparts (during merging we did create a clone of the original node hence for all old values there exists a counterpart in the `subgraphNode` graph). Does this description help? If yes, I will put it to the comment.